src/harness.js

```diff
@@ -655,8 +663,52 @@ class Harness {
       return contactSummaryFunction(resolvedContact, resolvedReports, resolvedLineage);
     }
   }
+
+  /**
+   * Evaluates a form's context expression (in conjuction with the person/place properties)
+   * @param {string} [form] Form that we want to test visibility
+   * @returns bool
+   */
+
+  async isFormVisible(form) {
```

Comment:
should there also be an `options` parameter here? `isFormVisible('myForm', { user: 'foo' });` i think this standard across other interfaces